### ISSUE TITLE
Create Function Score Query from multiple queries

### ISSUE THREAD

**Issue by trylika** [ongr-io/ElasticsearchBundle#236]
Currently we can only create FunctionScoreQuery() only from one query/filter, there is a need to pass multiple queries/filters

**Comment by saimaz** [comment]
There is implemented objective functions addition in #247 . I will gonna change the query addition to similar way. 

The plan is to behave like this:
```php
$term = new TermQuery('price', 5);
$queryString = new QueryStringQuery('name', 'Bob');

$functionScore = new FunctionScoreQuery();
$functionScore->addQuery($term);
$functionScore->addQuery($queryString);
...
```

From the `$search` object there is a getter to get all queries. 
https://github.com/ongr-io/ElasticsearchBundle/blob/1390c9b2e364bc37b2739eedb121d7015210a7e0/DSL/Query/QueryAwareTrait.php#L112

Is this will solve MatchMerge function score problem?

**Comment by saimaz** [comment]
Solved in #405

**Comment by saimaz** [comment]
Actually not in mentioned PR, it was moved to https://github.com/ongr-io/ElasticsearchDSL and solved there
